packages/sitecore-jss/src/i18n/graphql-dictionary-service.ts

```diff
@@ -49,13 +49,39 @@ const query = /* GraphQL */ `
   }
 `;
 
+const siteQuery = /* GraphQL */ `
+  query DictionarySiteQuery(
+    $siteName: String!
+    $language: String!
+    $pageSize: Int = 10
+    $after: String
+  ) {
+    site {
+      siteInfo(site: $siteName) {
+        dictionary(language: $language, first: $pageSize, after: $after) [
```

Comment:
It's impossible to use arrays in a query
Phrases should be fetched differently. Please, revisit internal docs
```
dictionary(language: $language, first: $pageSize, after: $after) {
    results {
      key
      value
    }
}
```

Reply:
That change in docs was just added today :)

Reply:
Yeah, this part - true, I didn't know
But it still wasn't possible to use arrays in a query =)